### PR TITLE
Adding Task documenter functionality.  

### PR DESCRIPTION
**Pull request by joe-belcan**
Known issues:

Documentation needs update to note that this is supported now but only for c style tasks.

packages/colibri/src/documenter/translation.json needs some help with its Russian translation of tasks.

versioning/auto testing magic that occurs in the workflows. add a test for tasks?

### CODE REVIEW

File: packages/colibri/src/config/config_web.ts [hunk 1]
```diff
@@ -616,7 +616,17 @@ export const WEB_CONFIG = `
               </select>
             </div>
           
-          
+            <div class="mb-3">
```
Comment (qarlosalberto) [inline packages/colibri/src/config/config_web.ts:622]: Have you used the Python script or manually?

File: packages/colibri/src/config/web_config.html [hunk 2]
```diff
@@ -594,6 +594,20 @@ <h6 class="card-subtitle mb-2 text-muted"></h6>
...
               </select>
             </div>
+
+
+
+            <div class="mb-3">
+              <label for="documentation-general-tasks" class="form-label">
+                Include tasks:
+                <span class="markConfig badge bg-secondary" id="mark_documentation-general-tasks"></span>
+              </label>
```
Comment (qarlosalberto) [inline packages/colibri/src/config/web_config.html:604]: Have you used the Python script or manually?

File: packages/colibri/src/documenter/translation.json [hunk 3]
```diff
@@ -111,6 +111,10 @@
         "english": "Functions",
         "russian": "Функции"
     },
+    "Tasks": {
+        "english": "Tasks",
+        "russian": "No Speaking Russian"
```
Comment (qarlosalberto) [inline packages/colibri/src/documenter/translation.json]: Just use Google translator.

**Review by qarlosalberto — CHANGES_REQUESTED** [review]

**Comment by qarlosalberto** [comment]
Thank you very much! Could you send me an email (carlosruiznaranjo@gmail.com) ? I want to talk with you.

File: packages/colibri/src/config/config_web.ts (hunk 1 above)
Comment (joe-belcan) [reply packages/colibri/src/config/config_web.ts:622]: This part was done by hand. was there a python script that can be used for these updates?
-edit I found the python script. trying to get it to run in windows

File: packages/colibri/src/config/web_config.html (hunk 2 above)
Comment (joe-belcan) [reply packages/colibri/src/config/web_config.html:604]: This part was done by hand. was there a python script that can be used for these updates?
-edit I found the python script. trying to get it to run in windows

File: packages/colibri/src/documenter/translation.json (hunk 3 above)
Comment (joe-belcan) [reply packages/colibri/src/documenter/translation.json]: Fixed

File: packages/colibri/src/config/config_web.ts (hunk 1 above)
Comment (joe-belcan) [reply packages/colibri/src/config/config_web.ts:622]: updated packages/colibri/src/config/helpers/configs/documentation.yml to get it to update correctly with running the python script.

**Comment by joe-belcan** [comment]
> Thank you very much! Could you send me an email ([carlosruiznaranjo@gmail.com](mailto:carlosruiznaranjo@gmail.com)) ? I want to talk with you.

I sent you an email. thanks!

**Comment by qarlosalberto** [comment]
thanks, good job!

**Review by qarlosalberto — APPROVED** [review]

**Pull request merged**